<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_newProcessClass</class>
 <widget class="QWidget" name="widget_newProcessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>113</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Black</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>widget_newProcess</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
QWidget{background:#2b2b2b}

QPushButton{border:none} 
QPushButton{color:#ffffff} 
QPushButton::hover{color:#faa028} 
QPushButton::pressed{color:#ffffff} 


QProgressBar {
border: none;
border-radius: 5px;

color: #1c1c1c;

background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,stop: 0 #f7efe6,stop: 0.3 #f7efe6,stop: 0.6 #f7efe6,stop: 1 #f7efe6);
}

QProgressBar::chunk {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,stop: 0 #f7cd94,stop: 0.3 #f7bb6a,stop: 0.6 #fcad44,stop: 1 #fa9c20  );

/*border: 0.5px solid #e17903;*/

/*
border: none;

border-bottom-right-radius: 5px;
    border-bottom-left-radius: 5px;
border-top-right-radius: 5px;
    border-top-left-radius: 5px;
}

QLineEdit{selection-background-color: #faa028; color:#ffffff;border:none }*/</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignRight|Qt::AlignTop">
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">
QWidget{background:#2b2b2b}

QPushButton{border:none} 
QPushButton{color:#ffffff} 
QPushButton::hover{color:#faa028} 
QPushButton::pressed{color:#ffffff} 


QProgressBar {
border: none;
border-radius: 5px;

color: #1c1c1c;

background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,stop: 0 #f7efe6,stop: 0.3 #f7efe6,stop: 0.6 #f7efe6,stop: 1 #f7efe6);
}

QProgressBar::chunk {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,stop: 0 #f7cd94,stop: 0.3 #f7bb6a,stop: 0.6 #fcad44,stop: 1 #fa9c20  );

/*border: 0.5px solid #e17903;*/


border: none;

border-bottom-right-radius: 5px;
    border-bottom-left-radius: 5px;
border-top-right-radius: 5px;
    border-top-left-radius: 5px;
}

QLineEdit{selection-background-color: #faa028; color:#ffffff;border:none }</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnCancel">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{color:#ffffff}



QWidget{background:#2b2b2b}

QPushButton{border:none} 
QPushButton{color:#ffffff} 
QPushButton::hover{color:#faa028} 
QPushButton::pressed{color:#ffffff} 


QProgressBar {
border: none;
border-radius: 5px;

color: #1c1c1c;

background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,stop: 0 #f7efe6,stop: 0.3 #f7efe6,stop: 0.6 #f7efe6,stop: 1 #f7efe6);
}

QProgressBar::chunk {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,stop: 0 #f7cd94,stop: 0.3 #f7bb6a,stop: 0.6 #fcad44,stop: 1 #fa9c20  );

/*border: 0.5px solid #e17903;*/


border: none;

border-bottom-right-radius: 5px;
    border-bottom-left-radius: 5px;
border-top-right-radius: 5px;
    border-top-left-radius: 5px;
}

QLineEdit{selection-background-color: #faa028; color:#ffffff;border:none }
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelbagName">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Current bag's name: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditBagName">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>14</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="textDirection">
         <enum>QProgressBar::BottomToTop</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item alignment="Qt::AlignLeft">
         <widget class="QLabel" name="labelRunning">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>running</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QPushButton" name="btnViewProcess">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>view</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
